server/database.js

```diff
@@ -0,0 +1,106 @@
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: '',
+  database: 'similarGames'
+});
+
+// const getGameSeriesID = (function(data, callback) {
+// const sql = "SELECT ID FROM series WHERE seriesName = data"
+// connection.query(sql, function(err, result, fields) {
+//   if (err)
+//     {
+//       console.log(err);
+//       callback(err, null);
+//     }
+//     else
+//     {
+//       console.log(result);
+//       callback(null, result);
+//     }
+// })
+// })
+
+const getGamesBySeries = (function(data, callback) {
+
+  const sql = "SELECT g.id, g.name, g.price, g.sale_percent, g.seriesID, g.releasedt, g.reviews, g.average_review, GROUP_CONCAT(t.tagName) as tags,  GROUP_CONCAT(m.imageFile) as media FROM (games AS g, gameTags AS gt, tags AS t, series AS s, media AS m) WHERE s.id = 3 AND g.id = gt.gameID AND gt.tagID = t.id AND m.gameID = g.id GROUP BY g.id, g.name;"
```

Comment:
At some point you may want to format your queries for readability - hit return before each new SQL syntax e.g. SELECT *blah blah blah* <hit return> GROUP_CONCAT if that makes sense.  Here's an example, hope this loads
<img width="495" alt="Screen Shot 2020-08-17 at 7 09 59 PM" src="https://user-images.githubusercontent.com/37882106/90453053-9915b980-e0bd-11ea-8bee-db03660cf682.png">